users/cameronjlarsen/features/caps_word.c

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021-2022 Google LLC
```

Comment:
Caps word should be removed, since it's a core feature, now.

https://docs.qmk.fm/#/feature_caps_word